src/app/leverage/components/stats/leverage-selector.tsx

```diff
@@ -0,0 +1,39 @@
+import { ChevronDownIcon } from '@heroicons/react/20/solid'
+
+type LeverageSelectorProps = {
+  leverage: string
+  leverageType: string
+  onClick: () => void
+}
+
+type CurrentLeverageProps = {
+  leverage: string
+}
+
+function CurrentLeverage(props: CurrentLeverageProps) {
+  return (
+    <div className='text-ic-gray-300 flex items-center gap-1 text-xs font-normal'>
+      <span>Current Leverage</span>
+      <span className='bg-ic-blue-300 text-ic-gray-950 rounded px-[6px] py-[1px] text-[9px] font-semibold'>
+        {props.leverage}
+      </span>
+    </div>
+  )
+}
```

Comment:
stylistic, but why not include in `<LeverageSelector />` below if it's just used once?

also could use `px-1.5` and `py-px`

Reply:
will keep separate until I know if it goes into the parent but merge otherwise. padding fixed.